ts/Core/Axis/LogarithmicAxis.ts

```diff
@@ -282,3 +287,13 @@ interface LogarithmicAxis extends Axis {
 LogarithmicAxis.compose(Axis); // @todo move to factory functions
 
 export default LogarithmicAxis;
+
+/**
+ * Activates rendering of negative logarithmic values.
+ *
+ * @type      {boolean}
+ * @since     8.1.0
+ * @apioption xAxis.allowNegativeLog
+ */
```

Comment:
Can we add a demo here? It's not like setting `allowNegativeLog` works out of the box, but some extra code is needed, right?

Reply:
I would expect this to work out of the box. Maybe we should adjust the usage of the `lin2log` function instead than having an other hackish sample. The `lin2log` and `log2lin` function are supposed to move to the composition instance, so this will break with HC 9.

Reply:
Previously it wasn't an API option, rather a plugin - so it was fine. With these changes perhaps we should probably provide full support then? Also, maybe it's a good idea to update related article with v9+ config?